<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageClearClass</class>
 <widget class="QWidget" name="ImageClearClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageClear</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Path">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Day">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Hour">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清理时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>天数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>小时</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Path">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>目录..</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_App">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exe">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>执行</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit_Time">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageClear.qrc"/>
 </resources>
 <connections/>
</ui>
